src/develop/pixelpipe_hb.c

```diff
@@ -983,7 +983,14 @@ static gboolean _transform_for_blend(const dt_iop_module_t *const self, const dt
 
 static dt_iop_colorspace_type_t _transform_for_picker(dt_iop_module_t *self)
 {
-  dt_iop_colorspace_type_t picker_cst = dt_iop_color_picker_get_active_cst(self);
+  // Special case the colorout module used for the global colorpicker. In this module
```

Comment:
Perhaps move this comment to `_request_color_pick`

Reply:
Done !